TUM Campus App/CardViewController.swift

```diff
@@ -72,13 +75,30 @@ extension CardViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setupLogo()
+        
+        if #available(iOS 11.0, *) {
+            self.navigationController?.navigationBar.prefersLargeTitles = true
+            self.navigationController?.navigationItem.largeTitleDisplayMode = .automatic
+        }
 
         refresh.addTarget(self, action: #selector(CardViewController.refresh(_:)), for: UIControlEvents.valueChanged)
         tableView.addSubview(refresh)
         tableView.tableFooterView = UIView(frame: CGRect.zero)
         tableView.separatorStyle = .none
         tableView.backgroundColor = .white
-        manager = (self.tabBarController as? CampusTabBarController)?.manager
+      
+        let storyboard = UIStoryboard(name: "CardView", bundle: nil)
+        guard let searchResultsController = storyboard.instantiateViewController(withIdentifier: "SearchResultsController") as? UIViewController else {
+            fatalError("Unable to instatiate a SearchResultsViewController from the storyboard.")
+        }
+        search = UISearchController(searchResultsController: searchResultsController)
+        search?.searchResultsUpdater = self
```

Comment:
I'd suggest that the search controller should also be the updater. That way we don't have to keep a reference to it...

Reply:
The search results table somehow becomes more laggy when the updater is the searchResultsController. I don't mind the reference if it gives us better performance.